client/src/views/TableDetail.vue

```diff
@@ -29,10 +31,17 @@ export default {
   props: ['workspace', 'table'],
   data () {
     return {
-      rowKeys:[]
+      rowKeys:[],
+      headers:[]
     }
+  },
+  computed: {    
+    
   },
   methods: {
+    classNameByIndex(index) {
```

Comment:
I would probably call this something more directly descriptive, like `rowClassName()` maybe.